spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -38,6 +38,30 @@
       end
     end
 
+    context 'when the limit is reached' do
```

Comment:
Maybe you can add a test for `create_list(:target, Target::MAX_TARGET_LIMIT -1)` and test that it allow to create the target.